src/PantherTestCaseTrait.php

```diff
@@ -161,12 +167,18 @@ protected static function createPantherClient(array $options = [], array $kernel
             }
         }
 
+        // Only allow browser arguments if a browser has been explicitly specified
+        if (\is_array($browserArguments) && !\array_key_exists('browser', $options)) {
+            @trigger_error('Specifying $browserArguments requires that you specifically set the browser in $options. $browserArguments will be ignored.', \E_USER_WARNING);
+            $browserArguments = [];
+        }
+
         self::startWebServer($options);
 
         if (static::CHROME === $browser) {
-            self::$pantherClients[0] = self::$pantherClient = Client::createChromeClient(null, null, $managerOptions, self::$baseUri);
+            self::$pantherClients[0] = self::$pantherClient = Client::createChromeClient(null, $browserArguments, $managerOptions, self::$baseUri);
```

Comment:
Can't we use the provided $options array to add an additional key for "browser_arguments", then this change won't be breaking.

Reply:
Hmm, I think I like this idea much better. Simpler, no deprecation warnings, we just transfer `$options['browser_arguments']` into that third argument and everything just works(tm)